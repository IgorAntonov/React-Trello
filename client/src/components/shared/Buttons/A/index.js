import styled from 'styled-components';
import PropTypes from 'prop-types';

import css from '../commonStyles';

const A = styled.a`
  ${css}
`;

A.propTypes = {
  href: PropTypes.string.isRequired
};

export default A;
