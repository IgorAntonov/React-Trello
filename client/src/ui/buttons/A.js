import styled from 'styled-components';
import PropTypes from 'prop-types';

import { styles } from './commonStyles';

export const A = styled.a`
  ${styles}
`;

A.propTypes = {
  href: PropTypes.string.isRequired,
  raised: PropTypes.bool,
  bgcolor: PropTypes.string
};
