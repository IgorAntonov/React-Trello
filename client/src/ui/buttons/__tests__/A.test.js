import React from 'react';
import { shallow } from 'enzyme';

import { A } from '../A';

describe('<A/>', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<A href="/" />);
  });

  test('renders', () => {
    expect(wrapper.exists()).toBe(true);
  });
  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
