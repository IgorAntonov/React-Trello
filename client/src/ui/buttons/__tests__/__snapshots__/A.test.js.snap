// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<A/> should match snapshot 1`] = `
<a
  className="A-bJefwH hzAmDC"
  href="/"
/>
`;
